docs/templates/plugin.rst.j2

```diff
@@ -424,19 +424,19 @@ Status
 {%   set module_states = { 'preview': 'not guaranteed to have a backwards compatible interface', 'stableinterface': 'guaranteed to have backward compatible interface changes going forward'} %}
 
 {%   if metadata %}
-{%     if metadata.status %}
+{%     if metadata.status|default(False) %}
 
 {%       for cur_state in metadata.status %}
 - This @{ plugin_type }@ is @{ module_states[cur_state] }@. *[@{ cur_state }@]*
 {%       endfor %}
 
 {%     endif %}
 
-{%     if metadata.supported_by %}
+{%     if metadata.supported_by|default(False) %}
 {%       set supported_by = support[metadata.supported_by] %}
 - This @{ plugin_type }@ is :ref:`maintained by @{ supported_by }@ <modules_support>`. *[@{ metadata.supported_by }@]*
 
-{%       if metadata.supported_by in ('core', 'network') %}
+{%       if metadata.supported_by|default(False) in ('core', 'network') %}
```

Comment:
At the docs meeting yesterday we agreed that metadata should just be removed from the docs altogether.  So just remove the block

Reply:
ok, i had understood we were removing from the plugins, but still allowing it's display, but i'll remove this from the template.